bcda/api.go

```diff
@@ -361,6 +361,50 @@ func serveData(w http.ResponseWriter, r *http.Request) {
 	http.ServeFile(w, r, fmt.Sprintf("%s/%s/%s", dataDir, jobID, fileName))
 }
 
+/*
+	swagger:route POST /auth/token auth getAuthToken
+
+	Get access token
+
+	Verifies parameters client_id and secret, and returns a JWT token that can be presented to the other API endpoints.
+
+	Consumes:
+	- application/x-www-form-urlencoded
+
+	Produces:
+	- application/json
+
+	Schemes: https
+
+	Responses:
+		200: tokenResponse
+		400: missingCredentials
+		401: invalidCredentials
+		500: serverError
+*/
+func getAuthToken(w http.ResponseWriter, r *http.Request) {
+	clientId := r.Header.Get("client_id")
+	secret := r.Header.Get("secret")
```

Comment:
This endpoint is supposed to be using basic authentication.

Reply:
Thanks for pointing that out.  Code updated to basic auth.